src/main/java/seedu/address/logic/parser/contactlistparsers/AddContactParser.java

```diff
@@ -31,19 +34,20 @@
      */
     public AddContactCommand parse(String args) throws ParseException {
         ArgumentTokenizer tokenizer =
-                new ArgumentTokenizer(args, PREFIX_NAME, PREFIX_EMAIL, PREFIX_TELEGRAM);
+                new ArgumentTokenizer(args, PREFIX_NAME, PREFIX_EMAIL, PREFIX_TELEGRAM, PREFIX_TAG);
         ArgumentMultimap argMultimap = tokenizer.tokenize();
 
-        if (!arePrefixesPresent(argMultimap, PREFIX_NAME)
+        if (!arePrefixesPresent(argMultimap, PREFIX_NAME, PREFIX_EMAIL)
                 || !argMultimap.getPreamble().isEmpty()) {
             throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddContactCommand.MESSAGE_USAGE));
         }
```

Comment:
I think we should check if all of the prefixes are present i.e. not just name and email but also telegram and tags? Since the Contact class requires all of the fields to be non-null?

In addition, I think inputting the command "addcontact n/mike e/example@web.com" without prefix telegram will cause below statement, 

`Telegram telegram = ParserUtil.parseTelegram(argMultimap.getValue(PREFIX_TELEGRAM).get());`

to throw a `NoSuchElementException` since `argMultimap.getValue(PREFIX_TELEGRAM)` will return an `Optional.empty`?

Reply:
This issue is actually being highlighted separately, in issue #376. That issue will modify Telegram to be an optional field and would ensure the above mentioned bugs would not be present. Thanks for the note! 👍

Reply:
Oh sorry, I didn't notice the issue earlier hahaha 😆 . Okay then!